docs/user_guide.md

```diff
@@ -58,9 +58,19 @@ The following screenshot show the "Settings" menu, where you can set workspace f
 
 <img src="../images/user_guide/create_workspace/workspace_settings.png" srcset="../images/user_guide/create_workspace/workspace_settings.png 2x" alt="create_workspace/WorkspaceSettings.png"/>
 
+## Browsing existing open source Widgets on GitHub
+
+Many widgets are freely available to download from GitHub. The following search query will retrieve all repositories tagged `wirecloud-widget`
+
+```text
+https://github.com/search?q=wirecloud-widget&type=Repositories
+```
+
+The [appendix](widgets.md) lists more details and links to the latest version of a series of over fifty open-source widgets available on GitHub.
+
 ## Browsing the Marketplace
 
-A mashup tool like WireCloud must support access to **marketplaces**, where people can offer and deal with services made
+A mashup tool like WireCloud also supports access to **marketplaces**, where people can offer and deal with services made
```

Comment:
So I think this change should be discarded.

Reply:
If this is to be _The recommended way of searching components_ then we'll need a re-wording - I guess a revert and a **link** to the _"getting latest bleeding edge development"_ section.

There needs to be a **fresh up-to-date marketplace** link as well - could you supply one?

Reply:
Fixed:  cca64bbc4